src/server/safeMutations/reactivateTeamMembersAndMakeNotifications.js

```diff
@@ -37,6 +37,10 @@ const reactivateTeamMembersAndMakeNotifications = async (invitees, inviter, team
   reactivatedUsers.forEach(({id: userId, tms}) => {
     getPubSub().publish(`${NEW_AUTH_TOKEN}.${userId}`, {newAuthToken: tmsSignToken({sub: userId}, tms)});
   });
+  reactivatedTeamMembers.forEach((teamMember) => {
+    const teamMemberAdded = {teamMember};
+    getPubSub().publish(`${TEAM_MEMBER_ADDED}.${teamId}`, {teamMemberAdded, ...subOptions});
```

Comment:
Are inactive team members actually removed from teams?

Reply:
we set an `isNotRemoved: false` flag on it. That way, archived projects still have an owner even when the person is long gone.